code-studio/src/js/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -93,6 +94,9 @@ var WorkshopForm = React.createClass({
 
   componentWillUnmount: function () {
     if (this.isGoogleMapsLoaded()) {
+      if (this.gm_authFailure) {
+        window.gm_authFailure = this.old_gm_authFailure;
```

Comment:
Does `old_gm_authFailure` need to be on window? Couldn't it just be a local variable in this module?